internal/cli/create/create_test.go

```diff
@@ -38,6 +38,7 @@ func TestCreateOutsideGoPath(t *testing.T) {
 	is.NoErr(td.Exists("go.sum"))
 	is.NoErr(td.Exists("package.json"))
 	is.NoErr(td.Exists("package-lock.json"))
+	is.NoErr(td.Exists("public/default.css"))
```

Comment:
I think we're missing some test cases still:
1. The scaffolded `public/default.css` gets deleted but views and public files still work as expected.
2. Someone brings their own `public/default.css` and that gets picked up and used.

Reply:
Added the test case in `public_test.go`